frontend/pages/policies/PolicyPage/PolicyPage.tsx

```diff
@@ -108,7 +102,7 @@ const PolicyPage = ({
         ? teamPoliciesAPI.load(policyTeamId, policyIdForEdit as number)
         : globalPoliciesAPI.load(policyIdForEdit as number),
     {
-      enabled: false,
+      enabled: !!policyTeamId,
```

Comment:
Having a false `enabled` flag is kind of confusing, because then you have to search for the refetch function to figure out where it's being called. Presumably it was not rendering immediately because it was waiting for the global config to apply, which is no longer needed. 

I was unable to produce any bugs after this change.